training/general/src/environments/sklearn-1.1-ubuntu20.04-py38-cpu/context/intel-extension/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM mcr.microsoft.com/azureml/openmpi4.1.0-ubuntu20.04:{{latest-image-tag}}
```

Comment:
Will you be updating in all other envs as well?

Reply:
@elesel This PR was work in progress. Was this closed by mistake? I would like to re-open it and work through the PR merge process.

Reply:
The PR was closed by GitHub as a side effect when I force pushed to all branches to remove some training data from git history. Would you please create a new PR from your existing branch?

Reply:
@elesel : I don't see a Re-open pull request option here. Does owner have the option to re-open?

Reply:
No, there's not an option to re-open this. Please create a new PR from the existing branch.

Reply:
i tried to create a new PR on my forked repo and pushed a small change to itn

I get this:
![image](https://user-images.githubusercontent.com/41214682/218559922-1fd27fbd-5d09-429f-9118-73b5c7930d9a.png)

Reply:
I'm sorry--I didn't notice that you were working from a fork. At this point I'd recommend ensuring you have a copy of your changes and then re-forking azureml-assets and re-applying your changes to the new fork.

Reply:
added a new PR- https://github.com/Azure/azureml-assets/pull/392 after re-forking and re-applying changes. Please review when you get a chance. Thanks.